src/main/java/edu/berkeley/cs/LinkStoreTitan.java

```diff
@@ -496,14 +528,68 @@ long getEdgeTime(Edge e) {
       tx.rollback();
       return 0;
     }
+    updateQueriesPerNode(getNodeId(src));
     long count = 0;
     Iterable<Edge> edges = src.getEdges(Direction.OUT);
     for (Edge edge : edges) {
+      updateQueriesPerEdge(edge.getLabel());
       if (edge.getLabel().compareToIgnoreCase(String.valueOf(link_type)) == 0) {
         count++;
       }
     }
     tx.commit();
     return count;
   }
+
```

Comment:
It looks like the statistics collection might add some overhead to the benchmark itself (not sure); perhaps add it as an optional thing?